plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/ListContainersParams.java

```diff
@@ -0,0 +1,160 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client.params;
+
+import org.eclipse.che.plugin.docker.client.json.Filters;
+
+import java.util.Objects;
+
+/**
+ * Arguments holder for {@link org.eclipse.che.plugin.docker.client.DockerConnector#listContainers(ListContainersParams)}.
+ *
+ * @author Alexander Andrienko
+ */
+public class ListContainersParams {
+    private Boolean all;
+    private Integer limit;
+    private String  since;
+    private String  before;
+    private Boolean size;
+    private Filters filters;
+
+    /**
+     * Adds parameter show all containers. Only running containers are shown by default.
+     *
+     * @param all
+     *         if all == true show all containers, if all = false show only running containers
+     */
+    public ListContainersParams withAll(Boolean all) {
```

Comment:
Avoid wrappers in parameters to prevent passing of nulls. Use wrappers in fields, getters only instead